examples/NavigationPlayground/package.json

```diff
@@ -14,7 +14,7 @@
     "expo": "^25.0.0",
     "react": "16.2.0",
     "react-native": "^0.52.0",
-    "react-navigation": "link:../.."
+    "react-navigation": "file:../.."
```

Comment:
@JulianKingman is it correct?

Reply:
@helielson npm doesn't support `link:`, so is yarn-only OK? If so, I'll change it back to `link:`
more here: https://github.com/codaxy/cxjs/issues/279#issuecomment-335096573

Reply:
indeed! 👍

Reply:
@helielson I changed it back to link, as the docs (https://reactnavigation.org/docs/guides/contributors#Run-the-Example-App) suggest using yarn.